src/adr/0022-cassandra-message-inconsistency.md

```diff
@@ -0,0 +1,88 @@
+# 22. Cassandra Message inconsistencies
+
+Date: 2020-02-27
+
+## Status
+
+Proposed
+
+## Context
+
+Messages are denormalized in Cassandra in order to:
+ - access them by their unique identifier (messageId), for example through the JMAP protocol
+ - access them by their mailbox identifier and Unique IDentifier within that mailbox (mailboxId + uid), for example 
```

Comment:
s/IDentifier/Identifier

Reply:
That's intended to explain what UID means (Unique IDentifier)